src/schema/v2/artwork/index.ts

```diff
@@ -213,6 +213,27 @@ export const ArtworkType = new GraphQLObjectType<any, ResolverContext>({
           }
         },
       },
+      editionSize: {
+        type: GraphQLString,
+        resolve: ({ edition_sets }) => {
+          if (edition_sets && edition_sets.length === 1) {
+            return edition_sets[0]!.edition_size
+          }
```

Comment:
Can maybe do 

```
if (editionSets?.length) {
  return edition_sets[0].edition_size
} 
```
Or even a one liner (since null / undefined is automatically handled):
```jsx
resolve: ({ edition_sets }) => edition_sets?.[0]?.edition_size
```

Reply:
💡 !